api/audit/models.py

```diff
@@ -72,17 +82,13 @@ def get_history_record_model_class(
         module = import_module(module_path)
         return getattr(module, class_name)
 
-    @hook(AFTER_SAVE)
+    @hook(
+        AFTER_SAVE,
+        priority=priority.HIGHEST_PRIORITY,
```

Comment:
Oh, cool - does this define the order in which hooks are run?

Reply:
Yep